actions/upload_sessions.go

```diff
@@ -215,3 +220,21 @@ func (usr *UploadSessionResource) CreateBeta(c buffalo.Context) error {
 	}
 	return c.Render(200, r.JSON(res))
 }
+
+
+func (usr *UploadSessionResource) GetPaymentStatus(c buffalo.Context) error {
+	session := models.UploadSession{}
+	err := models.DB.Find(&session, c.Param("id"))
+
+	if err != nil || session.ID == models.EmptyUUID {
```

Comment:
I think you can do `session == models.UploadSession{}` or `session.ID == uuid.Nil`.

Reply:
Sounds good, I chose the former so we dont have to import uuid